cmd/sidecars/utils.go

```diff
@@ -0,0 +1,33 @@
+package sidecars
+
+import (
+	"strings"
+
+	"github.com/inspr/inspr/pkg/meta"
+	"github.com/inspr/inspr/pkg/operator/k8s"
+	"github.com/inspr/inspr/pkg/utils"
+	corev1 "k8s.io/api/core/v1"
+)
+
+func generateReadPort() int32 { return int32(0) }
+
+func generateWritePort() int32 { return int32(0) }
```

Comment:
I would prefer a single function that returns a SidecarConnections object, containign both a read and write ports.